ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -517,3 +518,17 @@ export const applyDefaultStorage = ({ onFail, onSuccess, encrypt, encryptPasswor
             })
             .catch(onFail);
 };
+
+export const applyMountPointStorage = async ({ onFail, onSuccess }) => {
+    console.log("applyMountPointStorage");
+    await setInitializeLabelsEnabled({ enabled: true });
+    await setBootloaderDrive({ drive: "" });
+    const [partitioning] = await findPartitioning({ method: "MANUAL" });
+    const tasks = await partitioningConfigureWithTask({ partitioning });
+    runStorageTask({
+        task: tasks[0],
+        onFail,
+        onSuccess: () => applyPartitioning({ partitioning }).then(onSuccess)
+                .catch(onFail)
+    });
+};
```

Comment:
I don't know if it is on purpose, but unlike in applyDefaultStorageI think you are not catching failure of runStoragTask here. I am not really sure how much sense makes catching it though.